internal/graph/cached_resolver.go

```diff
@@ -38,6 +38,12 @@ var (
 		Name:      "check_cache_hit_count",
 		Help:      "The total number of cache hits for ResolveCheck.",
 	})
+
+	checkCacheInvalidHit = promauto.NewCounter(prometheus.CounterOpts{
+		Namespace: build.ProjectName,
+		Name:      "check_cache_invalid_hit_count",
+		Help:      "The total number of cache attempts which hit an invalid entry.",
+	})
```

Comment:
```suggestion
	checkCacheInvalidHit = promauto.NewCounter(prometheus.CounterOpts{
		Namespace: build.ProjectName,
		Name:      "check_cache_invalid_hit_count",
		Help:      "The total number of cache hits for ResolveCheck that were discarded because they were invalidated",
	})
```